cmd/nerdctl/network_prune.go

```diff
@@ -28,8 +28,11 @@ import (
 	"github.com/containerd/nerdctl/pkg/netutil/nettype"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
 )
 
+var dockerDefaultNetworkDrivers = []string{"host", "none", DefaultNetworkDriver} // exist by default
```

Comment:
`host` is not a default driver we should look for another adapted name

Reply:
?